source/mainnet/net/installation/downloads-testnet.rst

```diff
@@ -123,7 +123,7 @@ Linux-Docker
 
 Download the full suite for running a node on Linux using Docker. The suite contains Concordium Node, Concordium Client and cargo-concordium.
 
-   - `Download the Testnet suite for Linux <https://distribution.testnet.concordium.com/tools/linux/concordium-software-linux-4.1.1-1-testnet.tar.gz>`_
+   - `Download the Testnet suite for Linux <https://distribution.testnet.concordium.com/tools/linux/concordium-software-linux-4.2.1-1-testnet.tar.gz>`_
```

Comment:
This doesn't work at the moment. Will followup.

Reply:
@abizjak Any news on this?

Reply:
Not yet.

Reply:
The link should be https://distribution.testnet.concordium.com/tools/linux/concordium-software-linux-4.2.1-0-testnet.tar.gz

But @fdibat should inform you when this is built.